pages/osx/apachectl.md

```diff
@@ -0,0 +1,15 @@
+# apachectl
+
+> Apache HTTP Server control interface for macOS.
+
+- Load the org.apache.httpd launchd job:
+
+`apachectl start`
+
+- Unload the org.apache.httpd launchd job:
+
+`apachectl stop`
+
+- Unload, then loads the org.apache.httpd launchd job:
```

Comment:
You don't have to mention "org.apache.httpd" after the 2nd time. It is implicit.